.evg.yml

```diff
@@ -726,13 +753,13 @@ functions:
         TIMEOUT=1800  # 30 minute timeout
 
         while true; do
-            if grep -q "BUILD SUCCESSFUL" gradle_output.log; then
+            if grep -q "BUILD SUCCESSFUL" gradle_output.log 2>/dev/null; then
```

Comment:
Adding these here to clean up the log outputs:  `grep: gradle_output.log: No such file or directory`